dockerhq.sh

```diff
@@ -79,8 +106,31 @@ case $key in
     migrate)
         web_runner run --rm web python manage.py migrate $@
         ;;
-    runserver)
-        web_runner run --service-ports web $@
+    runserver-dev)
+	web_runner run --name commcarehq_web_1 --rm --service-ports web /mnt/docker/runserver_dev.sh
```

Comment:
That should only happen if the old container never got removed. Could be related to that compose bug you linked to further down?